rpcs3/rpcs3qt/game_list.cpp

```diff
@@ -37,6 +39,19 @@ void game_list::mouseMoveEvent(QMouseEvent *event)
 	m_last_hover_item = new_item;
 }
 
+void game_list::keyPressEvent(QKeyEvent* event)
+{
+	const auto modifiers = QApplication::keyboardModifiers();
+
+	if (modifiers & Qt::ControlModifier && event->key() == Qt::Key_F && !event->isAutoRepeat())
+	{
+		Q_EMIT FocusToSearchBar();
+		return;
```

Comment:
I think this is wrong, it probably needs to sethandled or something

Reply:
By default accepted property is set to true.